### PR TITLE
Fix icon option

### PR DESCRIPTION
**Pull request by EndrII**
This changes  fix an icon option for multi targets distributions.

### CODE REVIEW

**Comment by pzhlkj6612** [comment]
Hi. As there is no related issue or detailed commit message, I want to know what this PR does. That is, what is the difference between the behavior of CQtDeployer before and after this PR?

**Comment by EndrII** [comment]
There is a bug due to which the icon is not copyed in a multi-package distribution. At the moment, the error has not been fixed and it is not known where it is. But it reproduces with about a 10% chance.

In this pull request, I found the expected location where it could be and fixed the problem, but as it turned out, this is not the only place where the error occurs.

File: Deploy/configparser.cpp [hunk 1]
```diff
@@ -1548,7 +1548,18 @@ bool ConfigParser::smartMoveTargets() {
                                     QuasarAppUtils::Debug);
 
         auto pkgKey = i.value().getPackage();
-        valueLink(_config.packagesEdit(), pkgKey, DistroModule{pkgKey}).addTarget(newTargetKey);
+        if (!_config.packagesEdit().contains(pkgKey)) {
+
+            QuasarAppUtils::Params::log(QString("The target %0 belongs to a %1 package"
+                                                " but the %1 not inited!").
```
Comment (pzhlkj6612) [inline Deploy/configparser.cpp:1554]:
```suggestion
            QuasarAppUtils::Params::log(QString("The target %0 belongs to package %1"
                                                " but this package is not initialized!").
```

**Review by pzhlkj6612 — CHANGES_REQUESTED** [review]

File: Deploy/configparser.cpp (hunk 1 above)
Comment (EndrII) [reply Deploy/configparser.cpp:1554]: thanks

**Pull request merged**